README.md

```diff
@@ -91,7 +91,7 @@ terraform init -backend=false
 terraform plan -out=backend.plan -target=module.backend
 # Step 4: Apply the infrastructure plan
 terraform apply backend.plan
-# Step 5: Only after applying (building) the backend resources, write our terraform config
+# Step 5: Only after applying (building) the backend resources, write our terraform config. use the terraform apply output to find your kms key id, and substitude it into the command below.
```

Comment:
```suggestion
# Step 5: Only after applying (building) the backend resources, write our terraform config. Use the `terraform apply` output to find your kms key id, and substitute it into the command below.
```

Reply:
removed this in favor of simplified interface